app/src/main/java/com/example/themoviesdb/data/web/service/MovieService.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.themoviesdb.data.web.service
+
+import com.example.themoviesdb.domain.model.MoviesResponse
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface MovieService {
+
+    @GET("movie/now_playing")
+    suspend fun getMoviesNowPlaying(@Query ("api_key") api_key: String) : MoviesResponse
```

Comment:
according to Android conventions, we use camelCase to name the variables in the code, so please change api_key to apiKey